redfish/resource_redfish_virtual_media.go

```diff
@@ -144,6 +144,9 @@ func createRedfishVirtualMedia(service *gofish.Service, d *schema.ResourceData)
 
 	//Get terraform schema data
 	image := d.Get("image").(string)
+	if !strings.HasSuffix(image, ".iso") && !strings.HasSuffix(image, ".img") {
```

Comment:
These are the only 2 formats supported, please verify once so that any format is not missed

Reply:
The Map CD/DVD option will only work with a .ISO file. The .ISO file needs to be a bootable image for the server to boot correctly.

The Map Removable Disk option will only work with a .IMG file. The .IMG file needs to be a bootable image for server to boot correctly.